test/CrowdSalePermissioned.t.sol

```diff
@@ -43,7 +43,7 @@ contract CrowdSalePermissionedTest is Test {
         (bidder, bidderPk) = makeAddrAndKey("bidder");
         vm.startPrank(deployer);
 
-        auctionToken = new Molecules();
+        auctionToken = new IPToken();
         auctionToken.initialize("MOLECULES", "MOL-0001", Metadata(42, msg.sender, "ipfs://abcde"));
```

Comment:
IPToken, IPT-0001

Reply:
it also works fine like this :) and it's a nice nod to the past